src/Altinn.AccessManagement.Core/Helpers/RightsHelper.cs

```diff
@@ -50,5 +51,18 @@ public static List<AttributeMatch> GetDelegationSubjectAttributeMatches(Delegati
 
             return new();
         }
+
+        /// <summary>
+        /// Builds a RightsQuery request model for lookup of a users rights for a given resource registry service on behalf of the given reportee party
+        /// </summary>
+        public static RightsQuery GetRightsQueryForResourceRegistryService(int userId, string resourceRegistryId, int fromPartyId)
+        {
+            return new RightsQuery
+            {
+                To = new List<AttributeMatch> { new AttributeMatch { Id = AltinnXacmlConstants.MatchAttributeIdentifiers.UserAttribute, Value = userId.ToString() } },
+                From = new List<AttributeMatch> { new AttributeMatch { Id = AltinnXacmlConstants.MatchAttributeIdentifiers.PartyAttribute, Value = fromPartyId.ToString() } },
+                Resource = new AttributeMatch { Id = AltinnXacmlConstants.MatchAttributeIdentifiers.ResourceRegistryAttribute, Value = resourceRegistryId }.SingleToList()
```

Comment:
Foreløpig mest en kommentar til meg selv. Hvordan håndteres søk på gitt type

Reply:
Søk etter alle rettigheter man har for en gitt ResourceType? Er ikke noe vi har støtte for øyeblikket, men det er vel ikke værre enn at man utvider RightsQuery modellen med mulighet for å spesifisere type?